packages/api/src/hooks/useAllAccountsList.ts

```diff
@@ -0,0 +1,28 @@
+import useAccountsList from './useAccountsList';
+import useCFDAccountsList from './useCFDAccountsList';
+
+/** A custom hook to get all user accounts, including external apps accounts */
+const useAllAccountsList = () => {
+    const { data: accountsList, isError: isAccountsListError, isLoading: isAccountsListLoading } = useAccountsList();
+    const {
+        data: cfdsList,
+        isError: isCFDAccountsListError,
+        isLoading: isCFDAccountsListLoading,
+    } = useCFDAccountsList();
+
+    const accounts = {
+        wallets: accountsList?.filter(account => account.is_wallet),
+        dtrade: accountsList?.filter(account => account.is_trading),
+        ...cfdsList,
+    };
```

Comment:
```suggestion
    const accounts = {
        wallets: accountsList?.filter(account => account.is_wallet),
        dtrade: accountsList?.filter(account => account.is_trading),
        ...cfdsList,
    };
```
Best to memoize this, there is some filtering going on here

Reply:
Done